docs/sources/mimir/release-notes/v2.7.md

```diff
@@ -80,3 +80,4 @@ In Grafana Mimir 2.7, the following options, metrics, and labels have been remov
 - Ingester: track cortex_ingester_memory_series metric consistently with cortex_ingester_memory_series_created_total and cortex_ingester_memory_series_removed_total. [PR 4312](https://github.com/grafana/mimir/pull/4312)
 - Querier: fixed a bug which was incorrectly matching series with regular expression label matchers with begin/end anchors in the middle of the regular expression. [PR 4340](https://github.com/grafana/mimir/pull/4340)
 - Security: updated the Go version to 1.20.3 to fix CVE-2023-24538. [PR 4795](https://github.com/grafana/mimir/pull/4795)
+- For GEM 2.7 : The Deploy on Linux has the wrong URL for Download the binary section ( https://grafana.com/docs/enterprise-metrics/latest/setup/linux/#download-and-configure-the-gem-binary ). The correct URL should be https://dl.grafana.com/gem/releases/enterprise-metrics-v2.6.0-linux-amd64
```

Comment:
```suggestion
- For GEM 2.7 : The [Deploy GEM on a Linux host](https://grafana.com/docs/enterprise-metrics/latest/setup/linux/#download-and-configure-the-gem-binary) page contains an incorrect URL to the [Download and configure the GEM binary] section (https://grafana.com/docs/enterprise-metrics/latest/setup/linux/#download-and-configure-the-gem-binary). The correct URL is https://dl.grafana.com/gem/releases/enterprise-metrics-v2.6.0-linux-amd64.
```
Is `v2.6.0` indeed correct for the GEM 2.7 release?

Reply:
@charleskorn, not sure where that GEM issue is. My feedback belongs in the GEM repo.